tests/clone_test.go

```diff
@@ -698,8 +700,24 @@ var _ = Describe("[Serial]VirtualMachineClone Tests", Serial, func() {
 						libinstancetype.WaitForVMInstanceTypeRevisionNames(sourceVM.Name, virtClient)
 
 						if toRunSourceVM {
-							By("Starting the VM and expecting it to run")
-							sourceVM = RunVMAndExpectLaunchWithRunStrategy(virtClient, sourceVM, virtv1.RunStrategyAlways)
+							runStrategy := virtv1.RunStrategyAlways
+							sourceVM.Spec.RunStrategy = &runStrategy
```

Comment:
Please use `ptr.To()`
```suggestion
							sourceVM.Spec.RunStrategy = ptr.To(virtv1.RunStrategyAlways)
```

Reply:
Done :)